docarray/typing/tensor/image/abstract_image_tensor.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+
+from docarray.typing.tensor.abstract_tensor import AbstractTensor
+
+
+class AbstractImageTensor(AbstractTensor, ABC):
+    @abstractmethod
+    def bytes(self):
```

Comment:
why don't we add this `abstract` method directly to `AbstractTensor`? I do not see why this is related to `image` at all

Reply:
this one is specific to Images as it converts the tensor to a PNG image and then saves it. This is different from passing the tensor to bytes. 

I don't think that all of the Tensor should have a bytes method

Reply:
> as it converts the tensor to a PNG image and then saves it
The method name does not really convey that imo

Reply:
What name would you like ? Now it is `to_bytes`

Reply:
After checking the code i think the name is good, but it doesn't 'save' (to disk) anywhere, right?

Reply:
no no disk involved